js-miniapp-sample/src/pages/camera.js

```diff
@@ -87,6 +90,7 @@ const Camera = () => {
             data-testid="file-input-image-back"
             capture="environment"
             value={backCamera}
+            ref={cameraRef}
```

Comment:
I think the problem might be that you don't need to set `value={backCamera}` at all. The `input` should get the value set automatically by the browser after a file is chosen. At least that seems to be how it's working on the "File upload" screen:
https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-sample/src/pages/file-upload.js#L93

Reply:
thanks but yeah got it working in the end 😄